packages/larva-patterns/modules/story/story.center-vertical.js

```diff
@@ -0,0 +1,13 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const story_center_vertical = clonedeep( require( './story.vertical' ) );
+
+const horizontal_alignment = ' lrv-u-justify-content-center';
```

Comment:
These controller files should be kept as simple as possible - someday we will decide on a better syntax for them, and can create codemods or similar to replace with the new syntax. For that reason, add classes directly in the assignment to the object property vs. controlling redundancy with a variable, to keep the syntax as simple and consistent as possible.

A linter for these files to automate rules such as this that are currently in my brain only is part of the Larva Rollout, and now has a ticket - https://jira.pmcdev.io/browse/GUT-186 – but will be pretty low priority.